components/brave_rewards/content/rewards_service_impl.cc

```diff
@@ -236,12 +237,14 @@ void DeferCallback(base::Location location, Callback callback, Args&&... args) {
 // read comment about file pathes at src\base\files\file_path.h
 #if BUILDFLAG(IS_WIN)
 const base::FilePath::StringType kDiagnosticLogPath(L"Rewards.log");
+const base::FilePath::StringType kCreatorPrefixStore(L"RewardsCreators.db");
```

Comment:
Can we keep these in alphabetical order?

Reply:
Fixed.